libraries/AP_Scripting/docs/docs.lua

```diff
@@ -1774,6 +1779,11 @@ function relay:toggle(instance) end
 ---@return boolean
 function relay:enabled(instance) end
 
+-- return state of a relay
+---@param instance integer
+---@return uint8_t
+function relay:get(instance) end
```

Comment:
Whats the use case? I tend to think if you care this much you could just use the pin directly and bypass relay. You can do that now of course if you now which pin number to check.

Reply:
mission planner has buttons for relay control, with low/high/toggle. We need to be able to read these for scripting to act on these buttons